src/pt-server/decorators.ts

```diff
@@ -0,0 +1,21 @@
+export function logTime(_target: object, _funcName: PropertyKey, descriptor: PropertyDescriptor) {
```

Comment:
i think it would be better if you had allowed a logger to be injected here (maybe create a function that takes in a logger and returns the decorator, and then use the returned decorator when logging)